prac_05/state_names.py

```diff
@@ -4,7 +4,6 @@
 File needs reformatting
 """
```

Comment:
Please double check if you have reformatted state_names.py or not cause there is a comment about 'File needs reformatting' in doc string.

Reply:
ok will do , thanks alot